tldw_Server_API/app/api/v1/endpoints/media.py

```
@@ -1498,10 +1546,48 @@
             smj = _json.dumps(new_meta, ensure_ascii=False)
         except Exception:
             raise HTTPException(status_code=400, detail="safe_metadata is not JSON-serializable")
-        with db.transaction():
-            conn = db.get_connection()
-            conn.execute("UPDATE DocumentVersions SET safe_metadata=? WHERE id=? AND deleted=0", (smj, dv_id))
-            conn.commit()
+        with db.transaction() as conn:
+            # Bump version to satisfy sync trigger and refresh last_modified
+            try:
+                now_ts = datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%S.%f')[:-3] + 'Z'
+            except Exception:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)